lib/util.js

```diff
@@ -17,6 +17,17 @@ module.exports = {
 
   socketIo: function () {
     return socketIoEmitter(RedisClient.create())
+  },
+
+  mapLocaleToSendWithUS: function (locale) {
+    const lookup = {
+      'en': 'en-US',
```

Comment:
I see this here, but then that makes it confusing when you pass in en-US up above, since it will do the mapping here

Reply:
`mapLocaleToSendWithUS(user.get('settings').locale || 'en-US')` the || `en-US` is in case the locale is undefined out of settings